tutorials/terraform/sqoop/clusters-postgresql-data-proc-and-vm.tf

```diff
@@ -0,0 +1,257 @@
+# Infrastructure for Yandex Cloud Managed Service for PostgreSQL cluster.
+#
+# RU: https://cloud.yandex.ru/docs/managed-postgresql/tutorials/sqoop
+#
+# Set the configuration of the Managed Service for PostgreSQL cluster, Managed Service for Data Proc cluster and Virtual machine:
+locals {
+  folder_id           = ""      # Your folder ID.
+  network_id          = ""      # Network ID for Managed Service for PostgreSQL cluster, Data Proc cluster and VM.
```

Comment:
Network ID for the Managed Service for PostgreSQL cluster, Data Proc cluster, and VM.